apps/web/.env.development

```diff
@@ -33,12 +33,11 @@ API_HOST=http://localhost:3000
 
 # - - - - - - - - Auth - - - - - - - -
 
-AUTH_CLIENT_ID=<value goes here>
```

Comment:
I wouldn't remove these. Just so a dev knows what will need to be set in his .env.local to make it work.

Also if we are here, would you be so kinds to add a line in the readme explaining that for auth to work locally your newly env var needs to be turned on, and that these vars need to be set into .env.local?

Reply:
Done